extensions/arc/notebooks/arcDeployment/deploy.arc.control.plane.ipynb

```diff
@@ -0,0 +1,250 @@
+{
+    "metadata": {
+        "kernelspec": {
+            "name": "python3",
+            "display_name": "Python 3"
+        },
+        "language_info": {
+            "name": "python",
+            "version": "3.6.6",
+            "mimetype": "text/x-python",
+            "codemirror_mode": {
+                "name": "ipython",
+                "version": 3
+            },
+            "pygments_lexer": "ipython3",
+            "nbconvert_exporter": "python",
+            "file_extension": ".py"
+        }
+    },
+    "nbformat_minor": 2,
+    "nbformat": 4,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "![Microsoft](https://raw.githubusercontent.com/microsoft/azuredatastudio/main/extensions/arc/images/microsoft-small-logo.png)\n",
+                " \n",
+                "## Deploy Azure Arc Data Controller on an existing cluster\n",
+                " \n",
+                "This notebook walks through the process of deploying a <a href=\"https://azure.microsoft.com/services/azure-arc/\">Azure Arc controller</a> on an existing cluster.\n",
+                " \n",
+                "* Follow the instructions in the **Prerequisites** cell to install the tools if not already installed.\n",
+                "* The **Required information** will check and prompt you for password if it is not set in the environment variable. The password can be used to access the data controller.\n",
+                "\n",
+                "<span style=\"color:red\"><font size=\"3\">Please press the \"Run Cells\" button to run the notebook</font></span>"
```

Comment:
Just realized you have to update the text to "Run all" to reflect the recent ADS changes